packages/grafana-ui/src/components/Table/JSONViewCell.tsx

```diff
@@ -29,7 +29,7 @@ export function JSONViewCell(props: TableCellProps): JSX.Element {
   return (
     <div {...cellProps} className={inspectEnabled ? tableStyles.cellContainerNoOverflow : tableStyles.cellContainer}>
       <div className={cx(tableStyles.cellText, txt)}>
-        {!link && <div className={tableStyles.cellText}>{value}</div>}
+        {!link && <div className={tableStyles.cellText}>{displayValue}</div>}
```

Comment:
Displaying value causes an error because we display arrays and objects in the data inspector table